lms/djangoapps/commerce/utils.py

```diff
@@ -100,10 +100,17 @@ def get_checkout_page_url(self, *skus):
         Example:
             http://localhost:8002/basket/add/?sku=5H3HG5&sku=57FHHD
         """
-        return '{checkout_page_path}?{skus}'.format(
+        program_uuid = kwargs.get('program_uuid')
```

Comment:
If you are going to leave it this way, at least add something to the docstring above.